test/strongStartHsuTest.html

```diff
@@ -0,0 +1,506 @@
+<html><head> 
+    <title>SPOT Automated Testing</title>
+
+	<link rel="stylesheet" type="text/css" href="../spot.css">
+
+    <!--If running tests that do not have to do with the interface, run jsbuild.sh or jsbuild.bat
+        in root directory of the project. If running interface testing, run testbuild.sh
+        or testBuild.bat-->
+    <script src="../build/spot.js"></script>
+
+    <!--Mocha setup stuff (1/2)-->
+    <meta charset="UTF-8">
+    <script src="../lib/test/mocha.min.js"></script>
+    <script src="../lib/test/chai.min.js"></script>
+    <link rel="stylesheet" type="text/css" href="../lib/test/mocha.css">
+    <!--mocha setup stuff to be continued in document body-->
+    
+</head>
+<body style="padding-left: 5%; padding-right: 5%; padding-top: 20px">
+
+    <h2>SPOT Automated Testing</h2>
+
+    <!--Mocha stuff (2/2)-->
+    <script>
+        mocha.setup("bdd"); //brings "describe", "it", etc. into global namespace
+        //mocha.checkLeaks();
+        var assert = chai.assert; //no one wants to type out "chai.assert" every time
+    </script>
+    <div id="mocha">
+        StrongStart(k/p) Testing
+    </div>
+    <p>StrongStart(k/p) tableaux begin on page 95</p><p>https://ling.auf.net/lingbuzz/003512</p>
+    <!--Mocha is set up now, you just need to write and run tests-->
+    <script>
+
+        describe('Tableau (58) in Hsu 2016', function() {
+
+            it('Hsu_Tree1', function() {
+                assert.equal(strongStart_Hsu('', ptree2, 'syll','phi'), 1);
+                assert.equal(strongStart_Hsu('', ptree2, 'syll','i'), 0);
+            });
+            it('Hsu_Tree2', function() {
+                assert.equal(strongStart_Hsu('', ptree3, 'syll','phi'), 1);
+                assert.equal(strongStart_Hsu('', ptree3, 'syll','i'), 0);
+            });
+            it('Hsu_Tree3', function() {
+                assert.equal(strongStart_Hsu('', ptree4, 'syll','phi'), 1);
+                assert.equal(strongStart_Hsu('', ptree4, 'syll','i'), 0);
+            });
+            it('Hsu_Tree4', function() {
+                assert.equal(strongStart_Hsu('', ptree5, 'syll','phi'), 0);
+                assert.equal(strongStart_Hsu('', ptree5, 'syll','i'), 0);
+            });
+        });
+
+        describe('Tableau (59) in Hsu 2016', function() {
+
+            it('Hsu_Tree5', function() {
```

Comment:
For each it() statement, please include parenthesizeTree(whateverTheRelevantPtreeIs) in the description part.
e.g.: it('Hsu_Tree5: '+parenthesizeTree(ptree6), function(){...}) in this case.